### PR TITLE
Allow ramping up rtol in NonLinSIM

### PR DESCRIPTION
**Pull request by akva2**
This is a POC and not intended for merge as-is.

@kmokstad i need to know what a kosher version of this looks like.

### CODE REVIEW

**Comment by kmokstad** [comment]
Not sure when this is needed. If only in rare occasions, could it be implemented through a sub-class instead (we use to have a NSNonLinSim before, for instance), associated with that application only?

File: src/SIM/NonLinSIM.C [hunk 1]
```diff
@@ -190,6 +196,9 @@ ConvStatus NonLinSIM::solveStep (TimeStep& param, SolutionMode mode,
   if (solution.empty())
     return FAILURE;
 
+  if (rTolSteps > 0 && param.step == 1)
+    rTol *= std::pow(rTolStep, -rTolSteps);
```
Comment (kmokstad) [inline src/SIM/NonLinSIM.C]: At least, consider moving this statement into the parse-method (lines 114-117) then no need to check param.step == 1

**Comment by akva2** [comment]
kunne det vært en idè å skille ut dette (= sånn ca rtol, atol, dtol i NonLinSIM, NewmarkSIM, iterative lineærløsere) i et objekt, og gi en bruker-SIM mulighet til å instansere en versjon av denne klassen for bruk (med en enkel default) ?

**Comment by timovanopstal** [comment]
this pr was just open for the question...

**Pull request closed without merging**